src/main.c

```diff
@@ -0,0 +1,24 @@
+/* Copyright (C) 2020 Open Information Security Foundation
```

Comment:
I don't really understand why this patch is necessary. Can you explain?

Reply:
We want to compile suricata and the fuzz targets.
We want them to share as much code as possible.
Suricata and fuzz targets do not have the same `main` function

So, Suricata's `main` function must be isolated from the rest, in a file whose compiled object file will not be used by fuzz targets.

Other solutions were :
- to move away code from suricata.c to another file init.c : downside is git history problem
- to copy/paste init code from suricata.c to every fuzz target needing it : downside is discrepancy because of lack of code sharing

Another solution would be to move away `main` and command line parsing code to another file main.c (and not have `SC_main`)

Do you see any other way ? What do you prefer ?